ThirdModule/Models/Courier.cs

```diff
@@ -7,9 +7,6 @@
 namespace ThirdModule
 {
     public class Courier
```

Comment:
seems like Client and Courier are equal
but in real life no
so make them different
I mean add then list of orders or salary or address to Client